src/core/image/resize/metadata.handler.ts

```diff
@@ -0,0 +1,14 @@
+import { ImageService } from '../image.service';
+import type { FastifyInstance } from 'fastify';
+
+export const createMetadataHandler = (
+  path: string,
+  fastify: FastifyInstance,
+) => {
+  fastify.post(path, async (request) => {
+    const fileToProcess = await request.file();
+    const imageService = new ImageService();
+
+    return imageService.metadata(fileToProcess.file);
```

Comment:
Gdy zwracasz promisy o wiele lepiej jest robić to z wykorzystaniem `return await ...` w przypadku rejecta stack trace jest o wiele czytelniejszy.

Reply:
prawdopodobnie zmienię to gdy dorobię error handling